pkg/forwarder/forwarder_health.go

```diff
@@ -39,7 +39,7 @@ func initForwarderHealthExpvars() {
 }
 
 // forwarderHealth report the health status of the Forwarder. A Forwarder is
-// unhealthy if the API keys are not longer valid or if to many transactions
+// unhealthy if the API keys are not longer valid or if too many transactions
```

Comment:
the second part of the sentence is no longer true given your changes :)

Reply:
Fixed the language before the code, lol - thanks 🕵